lib/global-styles.php

```diff
@@ -495,13 +495,20 @@ function gutenberg_experimental_global_styles_merge_trees( $core, $theme, $user
 	$result = gutenberg_experimental_global_styles_normalize_schema( array() );
 
 	foreach ( array_keys( $core ) as $block_name ) {
-		foreach ( array( 'presets', 'styles', 'features' ) as $subtree ) {
+		foreach ( array( 'presets', 'features' ) as $subtree ) {
 			$result[ $block_name ][ $subtree ] = array_merge(
 				$core[ $block_name ][ $subtree ],
 				$theme[ $block_name ][ $subtree ],
 				$user[ $block_name ][ $subtree ]
 			);
 		}
+		foreach ( array( 'typography', 'color' ) as $subtree ) {
```

Comment:
Should we avoid a static list here and iterate over all keys instead?

Reply:
Nice suggestion https://github.com/WordPress/gutenberg/pull/24294/commits/4b9bc0ea87eda2692770ce816f79f1e0836d74a1